### PR TITLE
Page to list hazardsets

### PR DESCRIPTION
**Pull request by pgiraud**
Please review.

### CODE REVIEW

File: thinkhazard/templates/hazardsets.jinja2 [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<style>
```
Comment (arnaud-morvan) [inline thinkhazard/templates/hazardsets.jinja2]: Could be in admin folder and inherit admin base template

**Comment by arnaud-morvan** [comment]
I think you could add a pill in admin nav-bar see tamplates/admin/common.jinja2

**Comment by arnaud-morvan** [comment]
Note that I have reworked nav-bar in my current work on climate changes

**Comment by pgiraud** [comment]
New tab pill to be added later.

**Pull request merged**